chia/daemon/server.py

```diff
@@ -604,6 +608,39 @@ async def _state_changed(self, service: str, message: Dict[str, Any]):
     def state_changed(self, service: str, message: Dict[str, Any]):
         asyncio.create_task(self._state_changed(service, message))
 
+    async def _watch_log_file(self, fp: TextIO, loop: asyncio.AbstractEventLoop):
+        """
+        Loops every 5 seconds and reads in data from the log file and sends out update
+        Stops when there are no more websockets open for this service
+        """
+        await asyncio.sleep(5)
+        while True:
+
+            # Get all websockets opened for this particular service
+            websockets = self.connections.get(service_logs)
+            if websockets is None or not websockets:
+                fp.close()
+                return None
+
+            new_data: List[str] = await loop.run_in_executor(
+                io_pool_exc, fp.readlines, 1048576
```

Comment:
Isn't this 1MiB? Might want to define a constant, or do the 1024*1024*10 thing

;)

Reply:
Thanks - fixed with `10*1024*1024`